regional/main.tf

```diff
@@ -4,7 +4,7 @@
 resource "helm_release" "datadog_operator" {
   chart      = "datadog-operator"
   name       = "datadog-operator"
-  namespace  = kubernetes_namespace_v1.datadog.metadata[0].name
+  namespace  = "datadog"
```

Comment:
We are managing this in the onboarding code since we encounter some cyclic dependencies with our Istio and cert-manager workflows in the root modules.